Touch Bar Simulator/AppDelegate.swift

```diff
@@ -17,6 +19,28 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		$0.button!.toolTip = "Right-click or option-click for menu"
 	}
 
+	struct KeyboardShortcutRecorder: SwiftUI.View {
```

Comment:
This and `keyboardShortcutsVC` should be top-level.
```suggestion
	struct KeyboardShortcutRecorder: View {
```